dockers/docker-database-global/database_config.json

```diff
@@ -0,0 +1,18 @@
+{
+    "INSTANCES": {
+        "redis_global":{
+            "hostname" : "redis_global.server",
+            "port": 6385,
+            "unix_socket_path": "/var/run/redis/redis_global.sock",
+            "unix_socket_perm" : 777
+        }
+    },
+    "DATABASES" : {
+        "GLOBAL_APP_DB" : {
+            "id" : 8,
```

Comment:
Double check to make sure this ID doesn't conflict with the latest code.

Reply:
Does the ID even need to be unique? Check if DbConnector actually has a bug here that requires conflicts.

Reply:
8 current is not conflict with other. Yes, now db id has to be unique even crossing instances, this is a bug in sonic:
Duplicated dbid in redis multi-instances causes error #5351